tests/eips/eip4844/test_excess_data_gas.py

```diff
@@ -124,12 +128,10 @@ def block_base_fee() -> int:  # noqa: D103
 @pytest.fixture
 def env(  # noqa: D103
     parent_excess_data_gas: int,
-    parent_blobs: int,
     block_base_fee: int,
 ) -> Environment:
     return Environment(
-        excess_data_gas=parent_excess_data_gas,
-        data_gas_used=parent_blobs * DATA_GAS_PER_BLOB,
+        excess_data_gas=parent_excess_data_gas + TARGET_DATA_GAS_PER_BLOCK,
```

Comment:
Directly related to the comment on `common.py`, we can check if the `parent_blobs` parameter:
- If zero, `excess_data_gas=parent_excess_data_gas`
- else, `excess_data_gas=parent_excess_data_gas + TARGET_DATA_GAS_PER_BLOCK`